notification-extension/TestFile.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  TestFile.swift
+//  notification-extension
+//
+//  Created by Victor Babenko on 7/6/18.
+//  Copyright © 2018 Iterable. All rights reserved.
+//
+import Foundation
+
+class TestFile {
```

Comment:
It might be useful to add a short comment about why this file exists, to make sure we remember to delete it if we figure out a way to fix code coverage reports without this workaround.